### ISSUE TITLE
[bug] 加载中国地图时，右下角的小岛屿都被隐藏

### ISSUE THREAD

**Issue by muzlevy** [apache/echarts#18612]
### Version

5.0.2

### Link to Minimal Reproduction

No response

### Steps to Reproduce

1. 利用 `json`  数据来注册地图： `echarts.registerMap("xxx",json)` 
2. 然后加载

### 问题：渲染的 `canvas` 比较小的情况下，右下角部分岛屿会直接被隐藏；`canvas` 大一点的话就不会。这是什么引起的？可以解决吗？
如下图：
![image](https://github.com/apache/echarts/assets/72795965/e13ba215-5503-4d9b-835b-7e5edb949d90)

画布变大后显示完整了：
![image](https://github.com/apache/echarts/assets/72795965/a539d4b2-7edb-4b5d-83ac-d2f2a87d812e)

### Expected Behavior

no.

### Environment

```markdown
- OS:Ubuntu 20.04.6 LTS
- Browser: Chrome 111.0.5563.146（正式版本）
- Framework:
```

**Comment by helgasoft** [comment]
looks like subpixel issue to me. 
Maybe the real question is "How does ECharts handle subpixel display?". I suppose they are ignored (not shown).

**Comment by muzlevy** [comment]
> looks like subpixel issue to me. Maybe the real question is "How does ECharts handle subpixel display?". I suppose they are ignored (not shown).

Thank you very much, I can't express it well, this is what I want to ask.

**Comment by plainheart** [comment]
@muzlevy I can't reproduce this issue with this [example](https://codepen.io/plainheart/pen/ZEqjOPB). This issue should have been fixed by ecomfe/zrender#760. Instead of the v5.0.2, please try to use the newer versions >= v5.1.2.

**Comment by plainheart** [comment]
@muzlevy I can't reproduce this issue with this [example](https://codepen.io/plainheart/pen/ZEqjOPB). This issue should have been fixed by ecomfe/zrender#760. Instead of the v5.0.2, please try to use the newer versions >= v5.1.2.
